speedtest.sh

```diff
@@ -7,15 +7,25 @@ DB_HOST="${DB_HOST:-http://localhost:8086}"
 DB_NAME="${DB_NAME:-speedtest}"
 DB_USERNAME="${DB_USERNAME:-admin}"
 DB_PASSWORD="${DB_PASSWORD:-password}"
+SPEEDTEST_HOSTNAME="${SPEEDTEST_HOSTNAME}"
+SPEEDTEST_SERVER_ID="${SPEEDTEST_SERVER_ID}"
 
 run_speedtest()
 {
     DATE=$(date +%s)
     HOSTNAME=$(hostname)
+    CLI_OPTIONS=$1
 
     # Start speed test
-    echo "Running a Speed Test..."
-    JSON=$(speedtest --accept-license --accept-gdpr -f json)
+    if [ -z "$CLI_OPTIONS" ]; then
+      echo "Running a Speed Test with default host... "
+      JSON=$(speedtest --accept-license --accept-gdpr -f json)
+    else
+      echo "Running a Speed Test with options [$CLI_OPTIONS]... "
+      # shellcheck disable=SC2086
```

Comment:
shellcheck in my IDE is recommending to wrap the argument in double quotes
- https://github.com/koalaman/shellcheck/wiki/SC2086
- ![image](https://github.com/robinmanuelthiel/speedtest/assets/2152194/56f9416e-fff2-4df3-a214-915c4d8c6b51)

but when I do this, the option does not pass to the speedtest CLI correctly
```sh
docker run -e SPEEDTEST_SERVER_ID=30907 barryevans80/speedtest:wip
Running a Speed Test with options [-s 30907]...
speedtest: Missing argument for option: -s
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
parse error: Invalid numeric literal at line 1, column 10
./speedtest.sh: 31: arithmetic expression: expecting primary: "  / 125000 "
```